tornado/log.py

```diff
@@ -235,4 +244,13 @@ def define_logging_options(options=None):
     options.define("log_file_num_backups", type=int, default=10,
                    help="number of log files to keep")
 
+    options.define("log_file_rotating_when", type=str, default='midnight',
+                   help=("specify the type of TimedRotatingFileHandler interval "
+                         "other options:('S', 'M', 'H', 'D', 'W0'-'W6')"))
+    options.define("log_file_rotating_interval", type=int, default=1,
+                   help="The interval value of timed rotating")
+
+    options.define("rotating_mode", type=str, default='sized',
```

Comment:
All options defined in this file should start with `log`. Grammatically I would use `rotate` instead of `rotating`: `log_rotate_mode`, `log_rotate_interval`, `log_rotate_when`. I would also use `size` and `time` instead of `sized` and `timed`.